extensions/vscode/src/views/homeView.ts

```diff
@@ -1227,12 +1409,12 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
       ),
       commands.registerCommand(
         Commands.HomeView.ShowSelectConfigForDeployment,
-        this.showSelectConfigForDeployment,
+        this.showSelectOrCreateConfigForDeployment,
         this,
       ),
       commands.registerCommand(
         Commands.HomeView.CreateConfigForDeployment,
-        this.createConfigForDeployment,
+        this.showSelectOrCreateConfigForDeployment,
```

Comment:
I wonder if we should just combine the `package.json` commands here since these two now do the same thing.